includes/functions.php

```diff
@@ -229,6 +229,66 @@ function dokan_count_posts( $post_type, $user_id ) {
     return $counts;
 }
 
+/**
+ * Count stock product type from a user
+ *
+ * @global WPDB $wpdb
+ *
+ * @since DOKAN_LITE_SINCE
+ *
+ * @param string $post_type
+ * @param int    $user_id
+ * @param string $stock_type
+ *
+ * @return int $counts
+ */
+function dokan_count_stock_posts( $post_type, $user_id, $stock_type ) {
+    global $wpdb;
+
+    $cache_group = 'dokan_seller_product_data_' . $stock_type . '_' . $user_id;
```

Comment:
I don't see anywhere this cache has been deleted. Kindly delete cache when necessary